Annex60/Experimental/Benchmarks/AirFlow/Components/SimpleZone.mo

```diff
@@ -0,0 +1,116 @@
+    annotation (Placement(transformation(extent={{28,40},{48,60}})));
+  Modelica.Fluid.Interfaces.FluidPort_a port_a_vent(redeclare package Medium =
+        Medium)
+    annotation (Placement(transformation(extent={{-110,70},{-90,90}})));
+  Modelica.Thermal.HeatTransfer.Sources.PrescribedTemperature preTemp
+    "Dry bulb air temperature"
+    annotation (Placement(transformation(extent={{-60,-10},{-40,10}})));
+  BoundaryConditions.WeatherData.Bus weaBus annotation (Placement(
+        transformation(extent={{-20,80},{20,120}}), iconTransformation(extent={{
+            -128,30},{-108,50}})));
+equation
+  connect(weaBus.TDryBul, preTemp.T);
+  connect(conRoom.port_b, volRoom.heatPort) annotation (Line(
+      points={{0,0},{20,0}},
+      color={191,0,0},
+      smooth=Smooth.None));
+  connect(volRoom.ports[1], door.port_b2) annotation (Line(
+      points={{27.3333,-10},{27.3333,-34},{52,-34},{52,-6},{60,-6}},
+      color={0,127,255},
+      smooth=Smooth.None));
+  connect(volRoom.ports[2], door.port_a1) annotation (Line(
+      points={{30,-10},{32,-24},{44,-24},{44,6},{60,6}},
+      color={0,127,255},
+      smooth=Smooth.None));
+  connect(door.port_b1, port_a) annotation (Line(
+      points={{80,6},{86,6},{86,60},{100,60}},
+      color={0,127,255},
+      smooth=Smooth.None));
+  connect(door.port_a2, port_b) annotation (Line(
+      points={{80,-6},{86,-6},{86,-60},{100,-60}},
+      color={0,127,255},
+      smooth=Smooth.None));
+  connect(const.y, door.y) annotation (Line(
+      points={{49,50},{54,50},{54,0},{59,0}},
+      color={0,0,127},
+      smooth=Smooth.None));
+  connect(port_a_vent, volRoom.ports[3]) annotation (Line(
+      points={{-100,80},{-84,80},{-84,-54},{32.6667,-54},{32.6667,-10}},
+      color={0,127,255},
+      smooth=Smooth.None));
+  connect(preTemp.port, conRoom.port_a) annotation (Line(
+      points={{-40,0},{-20,0}},
+      color={191,0,0},
+      smooth=Smooth.None));
+  annotation (Diagram(coordinateSystem(preserveAspectRatio=false, extent={{-100,
+            -100},{100,100}}), graphics), Documentation(info="<html>
+<p>An air volume to represent a zone/room within a building that can be connected to a hallway element and to ventilation equipment. </p>
+<h4>Assumptions and limitations</h4>
+<p>This is a very simple room representation. The model is intended to roughly approximate a first order response of the zone to changes in outdoor air temperature. This is achieved by a thermal resistance in model conRoom and the capitancy of the mixing volume represented by the value for mSenFac. The G-Value of conRoom is approximated by the area of one outside wall multiplied with a U-Value of 1 W/(m**2*K). The value for mSenFac has been estimated from comparisons with other room models as shown in <a href=\"modelica://Annex60.Experimental.Benchmarks.AirFlow.Examples.ZoneStepResponse\">Annex60.Experimental.Benchmarks.AirFlow.Examples.ZoneStepResponse</a>.</p>
```

Comment:
Please use <i>1 W/(m<sup>2</sup> K)</i>  and use italics in general for numbers and <code> for variables such as mSenFac